DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/svd/src/svd.hpp

```diff
@@ -0,0 +1,273 @@
+#ifndef __SVD_HPP__
+#define __SVD_HPP__
+
+#include <sycl/ext/intel/ac_types/ac_complex.hpp>
+#include <sycl/ext/intel/ac_types/ac_int.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/sycl.hpp>
+#include <vector>
+
+#include "exception_handler.hpp"
+#include "memory_transfers.hpp"
+#include "non_std_convariance.hpp"
+#include "post_process.hpp"
+#include "streaming_eigen.hpp"
+#include "streaming_qrd.hpp"
+#include "svd_helper.hpp"
+#include "tuple.hpp"
```

Comment:
Since this design uses a mixture of header files that are unique to it, and header files from the shared `include` directory, you should separate them to make it clear which ones are shared and which are unique.